web-registry/src/components/organisms/ProjectTopSection.tsx

```diff
@@ -372,6 +345,7 @@ function ProjectTopSection({
               {landStory}
             </Description>
           )}
+          {isVcsProject && <AdditionalProjectMetadata metadata={metadata} />}
```

Comment:
I believe we should show this AdditionalProjectMetadata if at least one of the metadata field that we need to show there is present, not just the vcs project id
the side effect is that right now, the project activity is not appearing anymore on the kasigau page, while it should now appear in the additional metadata section

Reply:
I added a VCS project ID so metadata will show for Kasigau. Still a question about whether to handle non-VCS metadata the same way.

Reply:
sounds good